docs/content/getting-started/data-out/analyze-and-log.md

```diff
@@ -0,0 +1,86 @@
+---
+title: Analyze the data and log the results
+order: 3
+---
+
+
+
+In the previous sections, we explored our data and exported it to a Pandas dataframe. In this section, we will analyze the data to extract a "jaw open state" signal and log it back to the viewer.
+
+
+
+## Analyze the data
+
+This is admittedly not the most complicated part of this guide. We already identified that thresholding the `jawOpen` signal at 0.15 is all we need to produce a binary "jaw open state" signal.
```

Comment:
strange admission, but I appreciate the effort of trying to make it slightly self-ironic :)
I'd still leave it out because I'm struggling to find a formulation that doesn't sound odd to me

```suggestion
We already identified that thresholding the `jawOpen` signal at 0.15 is all we need to produce a binary "jaw open state" signal.
```